models/stocks.js

```diff
@@ -0,0 +1,60 @@
+const firestore = require('../utils/firestore')
+const stocksModel = firestore.collection('stocks')
+/**
+ * Adds Stocks
+ *
+ * @param stockData { Object }: stock data object to be stored in DB
+ * @return {Promise<{stockId: string}>}
+ */
+const addStock = async (stockData) => {
+  try {
+    const { id } = await stocksModel.add(stockData)
+    const { stockDetails } = await fetchStock(id)
+    return { id, stockDetails }
```

Comment:
Why are we making 2 DB calls? Can we just not return what we stored in the DB?

Reply:
Ok, let me check on this.

Reply:
https://github.com/Real-Dev-Squad/website-backend/pull/222/commits/c0e9f785e05885af8fd62562cbbcbd51626481bd

Reply:
Done.